e2e/validate-schema.e2e-spec.ts

```diff
@@ -215,3 +216,115 @@ describe('Validate OpenAPI schema', () => {
     });
   });
 });
+
+describe('Nested module scanning', () => {
```

Comment:
I have added the tests here for now, but if you prefer to separate them into a dedicated file, let me know and I will update accordingly.